src/pages/Profile.tsx

```diff
@@ -3,26 +3,61 @@ import { Link } from 'react-router-dom';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import PageContainer from '../components/layout/PageContainer';
+import Image from 'react-bootstrap/Image';
 import { Tagline } from '../components/layout/Titles';
+import styled from 'styled-components';
+import { colors, fonts } from '../theme/styleVars';
+import yellow_blob from '../images/yellow_blob_2.svg';
 
 const Profile: React.FC<{
   previewMode?: boolean;
 }> = ({ previewMode = false }) => {
   return (
     <PageContainer>
       <Row>
-        <Col lg={12}>
-          <Tagline>Your Profile</Tagline>
+        <Col lg={8}>
+          <Title>SIGN UP COMPLETED</Title>
+          <Tagline>Thanks for signing up!</Tagline>
           {previewMode && (
             <>
-              <p>Preview Mode</p>
-              <Link to="/sign-up-2">Continue to Sign Up 2</Link>
+              <Preview>
+                We’ll notify you via email once we’re ready to launch our beta
```

Comment:
Super minor thing, but replace the "special" quotes that come in from word processing docs (`’`) with `'` regular single quotes

Reply:
Sorry about that! Fixed.